packages/sdk/src/waku.ts

```diff
@@ -105,6 +109,8 @@ export class WakuNode implements Waku {
       this.relay
     );
 
+    this.health = getHealthManager();
```

Comment:
why is it here? I think it is not needed as `healthService` is exposed from `baseProtocol`

Reply:
agree with that